frontend/src/views/web/permission-denied.html

```diff
@@ -9,6 +9,13 @@
                     <div class="card-content">
                         <p><strong>Unable to create or select a team.</strong>
                             <br> {{perm.emailError}}</p>
+                            <br>
+                        <div ng-if="!perm.sendMail">
+                            <p> <span class="text-light-black">Didn't get the email?</span> <a class="waves-effect waves-dark dark-link ev-header-overflow" ng-click="perm.requestLink()">Resend Email</a>.</p>
+                        </div>
+                        <div ng-if="perm.sendMail">
```

Comment:
Will this also handle the case when verification email was sent a month ago and now the old verification link is expired. Will this check allow the user to request for a new verification email?

Reply:
It should work since the throttling rate in the backend is limited to 3/hour. If nothing has been made in the last 3 hours, a mail will be sent successfully and a 200 response is sent back.